app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java

```diff
@@ -104,7 +105,6 @@
 import com.nextcloud.talk.utils.rx.SearchViewObservable;
 import com.nextcloud.ui.popupbubble.PopupBubble;
 import com.yarolegovich.lovelydialog.LovelySaveStateHandler;
```

Comment:
The `lovelySaveStateHandler` field (and this import) is not needed anymore here.

Reply:
Eagle eyes! Removed it now, thanks! 🎉